src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -74,6 +76,7 @@ private static MeterProviderBuilder SetExporter(this MeterProviderBuilder builde
         switch (settings.MetricExporter)
         {
             case MetricsExporter.Prometheus:
+                Logger.Warning("Prometheus exporter is configured. It is intended for dev inner-loop, there is no plan to make it production ready.");
```

Comment:
```suggestion
                Logger.Warning("Prometheus exporter is configured. It is intended for the inner dev loop. Do NOT use in production");
```

I believe that we might add support for the AspNetCore version of the Prometheus exporter in the future.